src/DataWrangling/metadata.jl

```diff
@@ -67,17 +67,19 @@ function Metadatum(variable_name;
                    date = first_date(dataset, variable_name),
                    dir = default_download_directory(dataset))
 
-    date isa Union{CFTime.AbstractCFDateTime, Dates.AbstractDateTime} ||
-        throw(ArgumentError("date must be Union{Dates.AbstractDateTime, CFTime.AbstractCFDateTime}"))
-
     return Metadata(variable_name, dataset, date, dir)
 end
 
 # Just the current directory
-default_download_directory(dataset) = pwd()
+function default_download_directory end
```

Comment:
if this is not `pwd()` then delete the comment above

Reply:
ah yes, i removed this because the default should definitely not be current working

Reply:
Fair!